packages/perf-benchmarks/package.json

```diff
@@ -6,10 +6,10 @@
         "start": "best"
     },
     "devDependencies": {
-        "@best/cli": "4.0.0-beta5",
-        "@best/runner-headless": "4.0.0-beta5",
-        "@best/runner-remote": "4.0.0-beta5",
-        "@best/store-aws": "4.0.0-beta5",
+        "@best/cli": "4.0.0-beta6",
```

Comment:
Eugene pointed out that there was an issue with this version of best https://github.com/salesforce/lwc/pull/1982#discussion_r457875573

Reply:
True. FYI @diervo